Fix info files so that recent versions of install-info will be happier.

diff -uNr doc/krb425.info doc/krb425.info
--- doc/krb425.info	Fri Dec 17 15:52:38 1999
+++ doc/krb425.info	Tue Dec 21 14:28:13 1999
@@ -1,5 +1,8 @@
 This is Info file krb425.info, produced by Makeinfo version 1.68 from
 the input file krb425.texinfo.
+START-INFO-DIR-ENTRY
+* Krb425: (krb425).               Kerberos IV to Kerberos 5 Migration.
+END-INFO-DIR-ENTRY
 
 
 File: krb425.info,  Node: Top,  Next: Copyright,  Prev: (dir),  Up: (dir)
diff -uNr doc/krb5-admin.info doc/krb5-admin.info
--- doc/krb5-admin.info	Fri Dec 17 15:51:47 1999
+++ doc/krb5-admin.info	Tue Dec 21 14:28:15 1999
@@ -1,5 +1,8 @@
 This is Info file krb5-admin.info, produced by Makeinfo version 1.68
 from the input file admin.texinfo.
+START-INFO-DIR-ENTRY
+* Krb5-Admin: (krb5-admin).       Kerberos 5 Administrator's Guide.
+END-INFO-DIR-ENTRY
 
 
 Indirect:
diff -uNr doc/krb5-install.info doc/krb5-install.info
--- doc/krb5-install.info	Fri Dec 17 15:52:04 1999
+++ doc/krb5-install.info	Tue Dec 21 14:28:17 1999
@@ -1,5 +1,8 @@
 This is Info file krb5-install.info, produced by Makeinfo version 1.68
 from the input file install.texinfo.
+START-INFO-DIR-ENTRY
+* Krb5-Install: (krb5-install).   Kerberos 5 Installation Guide.
+END-INFO-DIR-ENTRY
 
 
 Indirect:
diff -uNr doc/krb5-user.info doc/krb5-user.info
--- doc/krb5-user.info	Fri Dec 17 15:52:34 1999
+++ doc/krb5-user.info	Tue Dec 21 14:28:05 1999
@@ -1,5 +1,8 @@
 This is Info file krb5-user.info, produced by Makeinfo version 1.68
 from the input file user-guide.texinfo.
+START-INFO-DIR-ENTRY
+* Krb5-User: (krb5-user).         Kerberos 5 User's Guide.
+END-INFO-DIR-ENTRY
 
 
 File: krb5-user.info,  Node: Top,  Next: Copyright,  Prev: (dir),  Up: (dir)
